dbms/src/Interpreters/Context.h

```diff
@@ -161,6 +161,7 @@ class Context
     {
         non_test,
         mpp_test,
+        cop_test,
```

Comment:
It looks like `mpp_test` and `cop_test` can be combined into one `mock_server_test`.